tools/build-custom-image.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+#Ultra custom image with custom cloud-fp, custom eap maven-plugin, custom wildfly maven plugin.
```

Comment:
when would we need both eap maven plugin *and* wildfly maven plugin?

Reply:
That is when new features appear in wildfly plugin prior any release. This script is really for ultra custom cases during WIP.